### ISSUE TITLE
Versions

### ISSUE THREAD

**Issue by ZM71** [Pyenb/macOS-ISOs#10]
does this have any beta version like Sequoia beta? i'd like Sequoia beta 2 iso

**Comment by Pyenb** [comment]
Sure, I had the Beta 1 ISO up, but removed it because I didn't think that anyone would want it. I can add it back later 👍

**Comment by ZM71** [comment]
i would like 1,2. Thanks. i mostly need the beta 2 as an ISO file.

**Comment by ZM71** [comment]
i made the beta with mist but i did not get an ISO file format. :(

**Comment by Pyenb** [comment]
Sorry for the long silence, I had some other things to do. I am also making some ISO's right now. When using mist you can select the "Export Types" and then I just select ISO.

**Comment by Pyenb** [comment]
But just asking. Are you able to see the Beta Versions in MIST? Even with the selector, they aren't showing up for me anymore.

**Comment by ZM71** [comment]
i cannot find the export type, could you help me make them?

**Comment by ZM71** [comment]
i've heard you can modify the config and log in using the latest beta-

**Comment by Pyenb** [comment]
> i cannot find the export type, could you help me make them?

![image](https://github.com/user-attachments/assets/fe9d7a97-eb59-400e-9616-d775a3c2574f)

![image](https://github.com/user-attachments/assets/a8fbb6dc-e5d4-4044-bba6-921716af0c41)

I always do it like this. Also yeah, it seems like when Apple releases the official versions, the beta channels get closed or whatever. I can still reupload the Beta 1 ISO, but would have to keep looking for the Beta 2 ISO.

**Comment by ZM71** [comment]
oh it was down there i am not used to this kind of file explorer so called finder, bruh i have to make it again.

**Comment by ZM71** [comment]
doesn't mist have beta 2 cause i'm not sure 1 has that log in feature but i can try.

**Comment by Pyenb** [comment]
> oh it was down there i am not used to this kind of file explorer so called finder, bruh i have to make it again.

Feel you. I am also no MacOS user an the GUI can be confusing at times.



> doesn't mist have beta 2 cause I'm not sure 1 has that log in feature but i can try.

Do you mean the 15.2 Beta or which one?

**Comment by ZM71** [comment]
the latest beta it is probably the 15.2

**Comment by ZM71** [comment]
yeah i usually get used to it quick but mac OS is quite different personally i hate and like the finder at the same time.

**Comment by Pyenb** [comment]
I added the 15.2 Beta. Will need some time for the torrent to pick up speed tho.
